_posts/2018-12-29-dask-on-hpc.md

```diff
@@ -0,0 +1,191 @@
+### 3.  Aimed for Scientific Processing
+
+In addition to being integrated with the Scipy and PyData software ecosystems,
+Dask is compatible with scientific data formats like HDF5, NetCDF, Parquet, and
+so on.  This is because Dask works with other libraries within the Python
+ecosystem, like Xarray, which already have strong support for scientific data
+formats and processing, and with C/C++/Fortran codes, such as is common for Python libraries.
+
+This native support is one of the major advantages that we've seen of Dask over Apache Spark.
+
+
+### 4.  Versatility of APIs
+
+And yet Dask is not designed for any particular workflow, but instead can
+provide infrastructure to cover a variety of different problems within an
+institution.  Many different kinds of workloads are possible:
+
+- You can easily handle Numpy arrays or Pandas Dataframes at scale, doing some numerical work or data analysis/cleaning,
+- You can handle any objects collection, like JSON files, text, or log files,
+- You can express more arbitrary task or job scheduling workloads with Dask Delayed, or real time and reactive processing with Dask Futures.
+
+Dask covers and simplifies many of the wide range of HPC workflows we've seen over the years.  Many workflows that were previously implemented using job arrays, simplified MPI (e.g. mpi4py) or plain bash scripts seem to be easier for our users with Dask.
+
+
+### 5.  Versatility of Infrastructure
+
+Dask is compatible with laptops, servers, HPC systems, and cloud computing.  The environment can change with very little code adaptation which reduces our burden to rewrite code as we migrate analysis between systems such as from a laptop to a supercomputer, or between a supercomputer and the cloud.
+
+```python
+# Local machines
+from dask.distributed import LocalCluster
+cluster = LocalCluster()
+
+# HPC Job Schedulers
+from dask_jobqueue import SLURMCluster, PBSCluster, SGECluster, ...
+cluster = SLURMCluster(queue='default', project='ABCD1234')
+
+# Hadoop/Spark clusters
+from dask_yarn import YARNCluster
+cluster = YarnCluster(environment='environment.tar.gz', worker_vcores=2)
+
+# Cloud/Kubernetes clusters
+from dask_kubernetes import KubeCluster
+cluster = KubeCluster(pod_spec={...})
+```
+
+Dask is more than just a tool to us; it is a gateway to thinking about a completely different way of providing computing infrastructure to our users. Dask opens up the door to cloud computing technologies (such as elastic scaling and object storage) and makes us rethink what an HPC center should really look like.
+
+
+### 6.  Cost and Collaboration
+
+Dask is free and open source, which means we do not have to rebalance our budget and staff to address the new immediate need of data analysis tools.
+We don't have to pay for licenses, and we have the ability to make changes to the code when necessary.  The HPC community has good representation among Dask developers.  It's easy for us to participate and our concerns are well understood.
```

Comment:
`It's easy for us to participate` I don't understand what you mean here

Reply:
For me it means we can easily ask questions, raise issues or propose some changes